otterdog/eclipse-opendut.jsonnet

```diff
@@ -19,6 +19,30 @@ orgs.newOrg('eclipse-opendut') {
     },
   },
   _repositories+:: [
+    orgs.newRepo('opendut') {
+      description: "Test Electronic Control Units around the world in a transparent network.",
+      private: true,
```

Comment:
are you sure you want to create a private repo?

Reply:
Yes. I'm going to switch to public when we are ready.

Reply:
In general thats not such a good idea. Private repos have some limitations that you need to be aware of while we are still on a free plan. Thats also quite unusual for doing open-source.

I would suggest that you create the repo as public, then you can already fork it, work on it in your fork, and once you think its worthwhile to merge it back to the upstream repo, create a PR with your changes.